.github/workflows/test.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         uses: Ana06/get-changed-files@v2.2.0 # it's a fork of jitterbit/get-changed-files@v1 which works better with pull requests
         with:
           format: 'json'
+        continue-on-error: true
```

Comment:
This sounds interesting, but the conditions below will still be falsy, example
```
contains(needs.detect_jobs_to_run.outputs.jobs, '-all-')
```
So let's merge https://github.com/prisma/prisma/pull/17598